tests/streamlit_ui_tests/streamlit.spec.ts

```diff
@@ -645,7 +650,11 @@ test.describe('Keyboard Shortcuts', () => {
     await expect(mito.locator('.index-header-selected').nth(1)).toHaveText('2');
   })
 
-  test('Select All', async ({ page }) => {
+  test.skip('Select All', async ({ page, browserName }) => {
+    if (browserName === 'webkit') {
```

Comment:
Webkit is Safari right? Did these tests catch the issue that these features aren't working on safari? cool!

Let's open an issue for them if so

Reply:
It's not really an issue, Safari just does not support these shortcuts.